### PR TITLE
chore: Remove get path

### PR DESCRIPTION
**Pull request by Jason-Wanxt**

### CODE REVIEW

**Review by chrstph-dvx — APPROVED** [review]

File: update_all [hunk 1]
```diff
@@ -15,7 +15,7 @@ yarn arbify --l2NetworkID 42161 --tokenList https://api.coinmarketcap.com/data-a
```
[the remark this answers is not in the document]
Comment (Jason-Wanxt) [reply update_all:15]: Ohhh, you are right, updated that file too, for this file, do we need to remove it? Or we can open a new pr for this one.

File: src/commands/full.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import { generateFullList } from '../lib/token_list_gen';
-import { writeToFile, getPath } from '../lib/store';
+import { writeToFile } from '../lib/store';
 import { Action, Args } from '../lib/options';
-import { ETHERSCAN_LIST_NAME } from '../lib/constants';
```
Comment (DZGoldman) [inline src/commands/full.ts:4]: can we remove ETHERSCAN_LIST_NAME from constants?

File: src/lib/token_list_gen.ts [hunk 3]
```diff
@@ -35,9 +35,8 @@ import { validateTokenListWithErrorThrowing } from './validateTokenList';
 import { constants as arbConstants } from '@arbitrum/sdk';
 import { readFileSync, existsSync } from 'fs';
 import { getNetworkConfig } from './instantiate_bridge';
-import { getPrevList, listNameToFileName } from './store';
+import { getPrevList } from './store';
 import { getArgvs } from './options';
-import { TOKENLIST_DIR_PATH } from './constants';
```
Comment (DZGoldman) [inline src/lib/token_list_gen.ts:40]: can we remove TOKENLIST_DIR_PATH from constants?

File: src/lib/token_list_gen.ts [hunk 4]
```diff
@@ -35,9 +35,8 @@ import { validateTokenListWithErrorThrowing } from './validateTokenList';
 import { constants as arbConstants } from '@arbitrum/sdk';
 import { readFileSync, existsSync } from 'fs';
 import { getNetworkConfig } from './instantiate_bridge';
-import { getPrevList, listNameToFileName } from './store';
```
Comment (DZGoldman) [inline src/lib/token_list_gen.ts:38]: can we remove listNameToFileName from store?

File: src/lib/store.ts [hunk 5]
```diff
@@ -38,10 +38,9 @@ export const getPath = (l1ListName: string) => {
 };
 
 export const getPrevList = (
-  l1ListName: string,
   arbifiedList: string | null,
 ): ArbTokenList | undefined => {
-  const path = arbifiedList ?? getPath(l1ListName);
```
Comment (DZGoldman) [inline src/lib/store.ts:44]: remove `getPath` funciton?

**Review by DZGoldman — CHANGES_REQUESTED** [review]
looks good - can clean / remove some more vars i think

**Review by chrstph-dvx — APPROVED** [review]

**Review by DZGoldman — APPROVED** [review]

**Pull request merged**